tests/md_test.py

```diff
@@ -167,11 +167,14 @@ def test_create_with_chunk_size(self):
         """Verify that it is possible to create and MD RAID with specific chunk size """
 
         with wait_for_resync():
-            succ = BlockDev.md_create("bd_test_md", "raid1",
-                                      [self.loop_dev, self.loop_dev2, self.loop_dev3],
-                                      1, None, True, 524288)
+            succ = BlockDev.md_create("bd_test_md", "raid0",
+                                      [self.loop_dev, self.loop_dev2],
+                                      0, None, chunk_size=512 * 1024)
```

Comment:
I have always tried to not rely on the python overrides in the tests. So please leave the ``True`` in place and pass just a number for chunk size (with an optional comment, of course).